teamA/jeongDoYoung/package.json

```
@@ -16,6 +16,7 @@
     "dbmate": "^1.0.3",
     "dotenv": "^16.0.3",
     "morgan": "^1.10.0",
+    "mysql": "^2.18.1",
     "typeorm": "^0.3.10",
     "mysql2": "^2.3.3"
```

mysql & mysql2 드라이버 패키지는 서로 동일한 기능을 하는 패키지 입니다. 따라서 둘 중 하나를 선택하여 설치를 진행해주세요.

이미 설치된 패키지나 모듈은 코드만 지우는 것이 아닌, 패키지 매니저 명령어를 통해 삭제해야합니다.

`npm uninstall PACKAGE_NAME`


```suggestion
    "mysql": "^2.18.1",
    "typeorm": "^0.3.10"
```